src/ovro_lwa_portal/viz/__init__.py

```
@@ -0,0 +1,146 @@
+from ovro_lwa_portal.viz._imports import check_viz_deps
+
+if TYPE_CHECKING:
+    import xarray as xr
+
+__all__ = [
+    "ImageExplorer",
+    "DynamicSpectrumExplorer",
+    "CutoutExplorer",
+    "SkyViewer",
+    "create_exploration_dashboard",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)